base/mpfr.jl

```diff
@@ -209,12 +209,43 @@ for (fJ, fC) in ((:si,:Clong), (:ui,:Culong))
 end
 
 function BigFloat(x::Float64, r::MPFRRoundingMode=ROUNDING_MODE[]; precision::Integer=DEFAULT_PRECISION[])
-    z = BigFloat(;precision=precision)
-    ccall((:mpfr_set_d, :libmpfr), Int32, (Ref{BigFloat}, Float64, MPFRRoundingMode), z, x, r)
-    if isnan(x) && signbit(x) != signbit(z)
-        z.sign = -z.sign
+    z = BigFloat(;precision)
+    if precision < 53
```

Comment:
Would be good to document why 53 is special here.

Reply:
will do. The reason is that if you have 53 or more bits, you don't have to deal with rounding which gets complicated and annoying.